### PR TITLE
dev-libs/libbulletml: don't override the system-wide flags

### PR DESCRIPTION
**Pull request by l29ah**

### CODE REVIEW

File: dev-libs/libbulletml/libbulletml-0.0.6.ebuild [hunk 1]
```diff
@@ -22,6 +22,9 @@ src_prepare() {
 	epatch "${FILESDIR}"/${P}-gcc43.patch \
 		"${FILESDIR}"/${P}-gcc46.patch
 	rm -r boost || die
+
+	# don't override the system-wide flags
+	sed -i -e '/^CFLAGS =/d;/^CXXFLAGS =/d' src/Makefile src/ygg/Makefile
```
Comment (a17r) [inline dev-libs/libbulletml/libbulletml-0.0.6.ebuild]: missing `|| die`

Comment (l29ah) [reply dev-libs/libbulletml/libbulletml-0.0.6.ebuild]: ok

**Comment by l29ah** [comment]
Yeah, this happens when half a year passes after you submit the pull request.

**Comment by a17r** [comment]
> Yeah, this happens when half a year passes after you submit the pull request.

Does it, though? Because I was curious as well, and the last commit to this whole directory happened on `Sat Dec 9 17:37:05 2017` so this change could not have worked at the start of the PR either. ;)

(sorry for the delay though; it is often hard to keep track of all the PR I have commented on)

**Pull request closed without merging**